packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1781,6 +1781,18 @@ export default class NewVm extends BaseComponent {
             <SectionContent>
               <Item label={_('secureBoot')}>
                 <Toggle onChange={this._toggleState('secureBoot')} value={secureBoot} />
+                {secureBoot && !pool.secureBootSetup && (
```

Comment:
`pool` can be `undefined` if it's a `self-service` user without permission on the `pool`.
In that case, an error `an error has occurred` will appears.